constants/exit_codes.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const (
+	EC_NoModFile = 15
```

Comment:
So tell me - what am I going to say about this variable name?

Now try to ask yourself that question before submitting the PR :p 

`ExitCodeNoModFile` please

And if we are defining consts for exit codes (a good idea), please raise an issue to use them consistently for all commands